.github/workflows/code_checks.yml

```
@@ -0,0 +1,53 @@
+name: code_checks
+on: [push]
+jobs:
+  linting:
+    runs-on: ubuntu-20.04
+    env:
+      RAILS_ENV: test 
+
+    services:
+      postgres:
+        image: postgis/postgis:11-2.5
+        env:
+          POSTGRES_USER: vets-api
+          POSTGRES_DB: vets-api_test
+          POSTGRES_PASSWORD: postgres
+        ports:
+          - 5432:5432
+
+      redis:
+        image: redis:3.2-alpine
+        ports: ['6379:6379']
+
+
+    steps:
+      - uses: actions/checkout@v2 #checkout step
+      
+      - uses: ruby/setup-ruby@v1
+
+      - name: Install dependencies (pdftk, poppler, imagemagick)
+        run: sudo apt install imagemagick pdftk poppler-utils
+ 
+      - name: Gem Cache
+        uses: actions/cache@v2
+        with:
+          path: vendor/bundle
+          key: gem-cache-${{ matrix.image }}-${{ hashFiles('Gemfile.lock') }}
+          
+      - name: Install Dependencies
+        run: bundle install --path vendor/bundle
+
+      - name: Run bundle-audit (checks gems for CVE issues)
+        run:  bundle exec bundle-audit check --update
+ 
+      - name: Run Rubocop
+        run: bundle exec rubocop --parallel --format progress --format json --out rubocop.json
+        
+      - name: Run Brakeman (static security analysis)
+        run:  bundle exec brakeman --ensure-latest --confidence-level=2 -o brakeman.html -o brakeman.text
```

I wonder if we should use https://github.com/reviewdog/action-brakeman instead.

```suggestion
      - uses: reviewdog/action-brakeman@v1
        with:
          github_token: ${{ secrets.github_token }}
```